sda-svc/templates/sftp-inbox-deploy.yaml

```diff
@@ -71,6 +71,8 @@ spec:
               key: mqPassword
         - name: BROKER_PORT
           value: {{ .Values.global.broker.port | quote }}
+        - name: BROKER_ROUTINGKEY
+          value: "inbox"
```

Comment:
shouldn't this come from `global.inbox.brokerRoutingKey`

Reply:
That should be fixed in sda-pipeline.